common/sgx/collateral.c

```diff
@@ -47,15 +47,15 @@ oe_result_t __oe_sgx_set_minimum_crl_tcb_issue_date(
 // which is 16 bytes long
 static oe_result_t _get_crl_ca_type(
     uint8_t* platform_instance_id,
-    oe_get_sgx_quote_verification_collateral_args_t* args)
+    uint8_t* collateral_provider)
```

Comment:
This is ok for now but this should probably use an enum type in the future instead of uint8_t.

Reply:
okay, currently there is no enum define for collateral_provider yet, need to do a refactor after this.